libgstc/c/libgstc_socket.c

```diff
@@ -70,13 +73,20 @@ create_new_socket (void)
 static GstcStatus
 open_socket (GstcSocket * self)
 {
+  int buffsize = GSTC_MAX_SOCKET_BUFFER_SIZE;
   gstc_assert_and_ret_val (NULL != self, GSTC_NULL_ARGUMENT);
 
   self->socket = create_new_socket ();
   if (-1 == self->socket) {
     return GSTC_SOCKET_ERROR;
   }
 
+  if (setsockopt (self->socket, SOL_SOCKET, SO_RCVBUF, &buffsize,
```

Comment:
I still don't understand the purpose of this. You increase the receive socket buffer size, but bellow we are only able to receive `GSTC_MAX_RESPONSE_LENGTH` so I feel this change has no effect. Am I missing something?

Reply:
I have changed it so that both match GSTC_MAX_RESPONSE_LENGTH